// [[Reason: undefined]] ////

//// [tests/cases/conformance/parser/ecmascript5/ErrorRecovery/ParameterLists/parserErrorRecovery_ParameterList1.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -1,19 +1,22 @@
 
 
 //// [parserErrorRecovery_ParameterList1.d.ts]
-declare function f(a: any, {}: {}): any;
+declare function f(a: any, {}: {}): invalid;
 /// [Errors] ////
 
 parserErrorRecovery_ParameterList1.ts(1,10): error TS2391: Function implementation is missing or not immediately following the declaration.
+parserErrorRecovery_ParameterList1.ts(1,10): error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
 parserErrorRecovery_ParameterList1.ts(1,19): error TS1005: ',' expected.
 parserErrorRecovery_ParameterList1.ts(2,6): error TS1005: ')' expected.
 
 
-==== parserErrorRecovery_ParameterList1.ts (3 errors) ====
+==== parserErrorRecovery_ParameterList1.ts (4 errors) ====
     function f(a: any {
              ~
 !!! error TS2391: Function implementation is missing or not immediately following the declaration.
+             ~
+!!! error TS9007: Declaration emit for this file requires type resolution. An explicit type annotation may unblock declaration emit.
                       ~
 !!! error TS1005: ',' expected.
     }: {}
          
